### PR TITLE
fix(workflow): Round metric alert interval

### PR DESCRIPTION
**Pull request by scttcper**
interval was being passed as "1.0m" and should be "1m"

### CODE REVIEW

File: src/sentry/incidents/charts.py [hunk 1]
```diff
@@ -189,7 +189,7 @@ def build_metric_alert_chart(
     aggregate = translate_aggregate_field(snuba_query.aggregate, reverse=True)
     # If we allow alerts to be across multiple orgs this will break
     project_id = snuba_query.subscriptions.first().project_id
-    time_window_minutes = snuba_query.time_window / 60
+    time_window_minutes = snuba_query.time_window // 60
```
Comment (evanpurkhiser) [inline src/sentry/incidents/charts.py:192]: Do we always want it floored? assuming yes

**Review by evanpurkhiser — APPROVED** [review]

File: src/sentry/incidents/charts.py (hunk 1 above)
Comment (scttcper) [reply src/sentry/incidents/charts.py:192]: yeah always

**Review by wedamija — APPROVED** [review]
Looks like `test_unfurl_metric_alerts_chart_crash_free` might be unstable. You might need to use `freeze_time` to help it out if you haven't already

File: tests/sentry/integrations/slack/test_unfurl.py [hunk 2]
```diff
@@ -282,12 +283,13 @@ def test_unfurl_metric_alerts_chart_transaction(self, mock_generate_chart):
         assert chart_data["incidents"][0]["id"] == str(incident.id)
 
     @patch("sentry.incidents.charts.generate_chart", return_value="chart-url")
+    @freeze_time("2022-02-21")
```
Comment (wedamija) [inline tests/sentry/integrations/slack/test_unfurl.py:286]: Actually looks like we already freeze time on the base class. I didn't realise the test fail was a real failure, you can probably just remove this.

**Pull request merged**